client/app/views/admin/users.html

```diff
@@ -162,7 +162,8 @@
                   </div>
                   <div class="form-group">
                     <label data-translate>Password</label>
-                    <input name="password" class="form-control" data-ng-model="user.password" maxlength="{{node.maximum_namesize}}" type="password" />
+                    <input name="password" class="form-control" data-ng-model="user.password" maxlength="{{node.maximum_namesize}}" type="password" zxcvbn="passwordStrength" />
```

Comment:
4 is the maximum rank gaved by zxcvbn.

we evaluated to accept rank 3 and 4 that are the current acceptable and strong levels